### ISSUE TITLE
Create a recording of a demo (and keep it up-to-date)

### ISSUE THREAD

**Issue by simonswine** [grafana/tns#25]
I think it would be helpful to have a quick GIF demo of the project.

I have recorded this example (using https://gifcap.dev/), let me know what you think?

A comment from @tomwilkie was, that I should eventually use the `|= "error"` syntax to find the correct log lines.

![recording](https://user-images.githubusercontent.com/223048/87411591-db7c4000-c5bf-11ea-9f9c-347ca9df7e66.gif)

/cc @jdbaldry

**Comment by jdbaldry** [comment]
Yeah looks great! Really good pacing considering it's a longer gif too. 
I agree with Tom that the last step should be `{pod="xxx"} |= "error"` and then it's pretty much exactly as I had it demoed to me

**Comment by tomwilkie** [comment]
Looks good! @davkal @aocenas would be good to do something about that split view - detect when the screen is narrow and split vertically (or don't split?  or leave as an option on the link?)

**Comment by aocenas** [comment]
I think technically, making it responsive so at some sizes the splits will just vertically stack may not be difficult. But I have a feeling that most feedback I heard was that people prefer opening the link as normal link, so maybe opening in a split should be a special non default action (like requiring a modifier key or on hover option or something like that.)
